File: api/src/main/java/org/algo/invest/core/quotes/EnergyQuotes.java
package org.algo.invest.core.quotes;

import java.util.LinkedHashMap;
import java.util.Map;

import org.algo.invest.model.QuoteRecord.Industry;
import org.algo.invest.model.QuoteSymbolMetaData;

public class EnergyQuotes {

	public static Map<String, QuoteSymbolMetaData> getQuotes(){
		
		Map<String, QuoteSymbolMetaData> symbolNameMapping = new LinkedHashMap<String, QuoteSymbolMetaData>();
		
		// Energie
		symbolNameMapping.put("RWE.DE", new QuoteSymbolMetaData("RWE.DE","RWE","703712","",Industry.ENERGY));
		symbolNameMapping.put("ED", new QuoteSymbolMetaData("ED","Consolidated Edison","911563","",Industry.ENERGY));
		symbolNameMapping.put("BLDP", new QuoteSymbolMetaData("BLDP","Ballard Power Systems","A0RENB","",Industry.ENERGY));
		symbolNameMapping.put("D7G.F", new QuoteSymbolMetaData("D7G.F","Nel Asa","A0B733","",Industry.ENERGY));		
		symbolNameMapping.put("27W.F", new QuoteSymbolMetaData("27W.F","Powercell","A14TK6","",Industry.ENERGY));
		symbolNameMapping.put("PLUG", new QuoteSymbolMetaData("PLUG","Plug Power","A1JA81","",Industry.ENERGY));
		
		// Erneuerbare Energien
		symbolNameMapping.put("NEE", new QuoteSymbolMetaData("NEE","Nextera Energy","A1CZ4H","",Industry.RENEWABLE_ENERGY));
		symbolNameMapping.put("D2G.F", new QuoteSymbolMetaData("D2G.F","Ørsted","A0NBLH","",Industry.RENEWABLE_ENERGY));
		symbolNameMapping.put("VWS.DE", new QuoteSymbolMetaData("VWS.DE","Vestas Wind Systems","A0MRJJ","",Industry.RENEWABLE_ENERGY));
		symbolNameMapping.put("SEDG", new QuoteSymbolMetaData("SEDG","Solaredge","A14QVM","",Industry.RENEWABLE_ENERGY));
		symbolNameMapping.put("CAP.DE", new QuoteSymbolMetaData("CAP.DE","Encavis","609500","",Industry.RENEWABLE_ENERGY));
		symbolNameMapping.put("ENPH", new QuoteSymbolMetaData("ENPH","Enphase Energy","A1JC82","",Industry.RENEWABLE_ENERGY));
		symbolNameMapping.put("CXGH.F", new QuoteSymbolMetaData("CXGH.F","Goldwind","A1C0QD","",Industry.RENEWABLE_ENERGY));
		
		// Akku
		symbolNameMapping.put("VAR1.DE", new QuoteSymbolMetaData("VAR1.DE","Varta","A0TGJ5","",Industry.AKKU));
		symbolNameMapping.put("BY6.F", new QuoteSymbolMetaData("BY6.F","BYD","A0M4W9","",Industry.AKKU));
		
		return symbolNameMapping;
	}
}

File: api/src/main/java/org/algo/invest/core/quotes/energyQuotes.kt
package org.algo.invest.core.quotes

import org.algo.invest.model.SubIndustry
import org.algo.invest.model.QuoteSymbolMetaData
import java.util.LinkedHashMap
import org.algo.invest.model.Industry

val energyQuotes: Map<String, QuoteSymbolMetaData>
    get() {
        val symbolNameMapping: MutableMap<String, QuoteSymbolMetaData> = LinkedHashMap()

        symbolNameMapping["RWE.DE"] = QuoteSymbolMetaData("RWE.DE", "RWE", "703712", Industry.ENERGY, SubIndustry.CONVENTIONAL)
        symbolNameMapping["ENR.DE"] = QuoteSymbolMetaData("ENR.DE", "Siemens Energy", "ENER6Y", Industry.ENERGY, SubIndustry.CONVENTIONAL)
        symbolNameMapping["ED"] = QuoteSymbolMetaData("ED", "Consolidated Edison", "911563", Industry.ENERGY, SubIndustry.CONVENTIONAL)

        symbolNameMapping["NEE"] = QuoteSymbolMetaData("NEE", "Nextera Energy", "A1CZ4H", Industry.ENERGY, SubIndustry.RENEWABLE)
        symbolNameMapping["D2G.F"] = QuoteSymbolMetaData("D2G.F", "Ørsted", "A0NBLH", Industry.ENERGY, SubIndustry.RENEWABLE)
        symbolNameMapping["VWS.DE"] = QuoteSymbolMetaData("VWS.DE", "Vestas Wind Systems", "A0MRJJ", Industry.ENERGY, SubIndustry.RENEWABLE)
        symbolNameMapping["SEDG"] = QuoteSymbolMetaData("SEDG", "Solaredge", "A14QVM", Industry.ENERGY, SubIndustry.RENEWABLE)
        symbolNameMapping["GTQ1.DE"] = QuoteSymbolMetaData("GTQ1.DE", "Siemens Gamesa", "A0B5Z8", Industry.ENERGY, SubIndustry.RENEWABLE)
        symbolNameMapping["ECV.DE"] = QuoteSymbolMetaData("ECV.DE", "Encavis", "609500", Industry.ENERGY, SubIndustry.RENEWABLE)
        symbolNameMapping["ENPH"] = QuoteSymbolMetaData("ENPH", "Enphase Energy", "A1JC82", Industry.ENERGY, SubIndustry.RENEWABLE)
        symbolNameMapping["CXGH.F"] = QuoteSymbolMetaData("CXGH.F", "Goldwind", "A1C0QD", Industry.ENERGY, SubIndustry.RENEWABLE)
        symbolNameMapping["BLDP"] = QuoteSymbolMetaData("BLDP", "Ballard Power Systems", "A0RENB", Industry.ENERGY, SubIndustry.RENEWABLE)
        symbolNameMapping["D7G.F"] = QuoteSymbolMetaData("D7G.F", "Nel Asa", "A0B733", Industry.ENERGY, SubIndustry.RENEWABLE)
        symbolNameMapping["27W.F"] = QuoteSymbolMetaData("27W.F", "Powercell", "A14TK6", Industry.ENERGY, SubIndustry.RENEWABLE)
        symbolNameMapping["PLUG"] = QuoteSymbolMetaData("PLUG", "Plug Power", "A1JA81", Industry.ENERGY, SubIndustry.RENEWABLE)

        symbolNameMapping["VAR1.DE"] = QuoteSymbolMetaData("VAR1.DE", "Varta", "A0TGJ5", Industry.ENERGY, SubIndustry.AKKU)
        symbolNameMapping["BY6.F"] = QuoteSymbolMetaData("BY6.F", "BYD", "A0M4W9", Industry.ENERGY, SubIndustry.AKKU)

        return symbolNameMapping
    }